File: Modul 6/Soal 3/PRAK503-2210817310006-Ahmad Muhsin Zein.py
def maksimal (a, b):
    if a > b : return a
    else : return b

def minimal (a, b) :
    if a < b : return a
    else : return b

batas = 0; maks = -100000; minim = 100000
bilangan = int(input())
while batas < bilangan :
    nilai = map(int,input().split())
    for angka in nilai :
        maks = maksimal(maks, angka)
        minim = minimal(minim, angka)
        batas += 1
print(maks, minim)

File: Modul 6/Soal 3/PRAK503-2210817310006-Ahmad Muhsin Zein.c
int maksimal (int a,int b){
if(a > b){return a;}
else{return b;}}

int minimal (int a,int b){
if(a < b){return a;}
else{return b;}}

int main(){
int batas,bilangan,maks,minim,nilai;
batas = 0; maks = -100000; minim = 100000;
scanf("%d", &bilangan);
while (batas < bilangan){
    scanf ("%d", &nilai);
    maks = maksimal(maks, nilai);
    minim = minimal(minim, nilai);
    batas ++;
}
printf("%d %d",maks, minim);
}
